owncloudApp/src/main/java/com/owncloud/android/ui/activity/ReceiveExternalFilesActivity.java

```diff
@@ -161,6 +163,8 @@ public class ReceiveExternalFilesActivity extends FileActivity
 
     private static final String DIALOG_WAIT_COPY_FILE = "DIALOG_WAIT_COPY_FILE";
 
+    Pattern pattern = Pattern.compile("[/\\\\]");
```

Comment:
This is Java but, can we follow a similar approach as before so that we can avoid regex?

Reply:
The option I see in Java would be to create an array with those two values ​​and then loop through the characters. I think the current option is cleaner using a pattern. In Kotlin it is easy to do it like this because it is cleaner, but in Java, I see it as more cumbersome. So I would leave it as it is.